.typos.toml

```diff
@@ -24,13 +24,15 @@
 "vertx" = "vertx"
 
 [files]
-ignore = [
-    "**/cobertura.ser"
+extend-exclude = [
+    "**/cobertura.ser"  # 排除该路径
```

Comment:
I think we can remove this chinese annotation

Reply:
Deleted